plugins/scaffolder-backend/src/service/router.test.ts

```diff
@@ -267,105 +295,74 @@ describe('createRouter', () => {
       );
     });
 
-    it('should not throw when an invalid authorization header is passed', async () => {
```

Comment:
@benjdlambert, I think this test falls into the realm of the identity api implementation now with my changes? The DefaultIdentiyClient is returning undefined from getIdentity if the request cannot be authenticated or fails in authentication (this point can and probably should be debated).

Thoughts?